content/4_Gold/Modular.mdx

```diff
@@ -479,6 +479,148 @@ int main() {
 
 </CPPOnly>
 
+## Chinese Remainder Theorem
```

Comment:
follow module format (aka topic header should only be 1 hashtag)

Reply:
All the topic headings in the module start with a double. I'll reduce all by one.

Reply:
Should I reduce all by one ?

Reply:
yeah

Reply:
Done